File: Desktop/Python/Exercises/python/7.py
alpha = "abcdefghijklmnopqrstuvwxyz"
from datetime import datetime

input("Hit ENTER when ready ")
seconds1 = datetime.now().second
useralpha = input()
seconds2 = datetime.now().second

if useralpha == alpha:
    print(seconds2 - seconds1)
else:
    print("You got it wrong, try again")

File: Desktop/Python/Exercises/rust/7.rs
use std::{io, time::Instant};

fn main() {
    let mut alpha = String::new();
    println!("Hit ENTER when ready");
    io::stdin().read_line(&mut alpha).expect("Failed to read line");
    let start = Instant::now();
    println!("Type the alphabet!");
    io::stdin().read_line(&mut alpha).expect("Failed to read line");
    if alpha.trim() == String::from("abcdefghijklmnopqrstuvwxyz") {
        println!("Correctly typed alphabet in {:?}", start.elapsed());
    } else {
        println!("You didn't type the alphabet correctly")
    }
}
